pymc4/distributions/tensorflow/distribution.py

```diff
@@ -0,0 +1,23 @@
+from ..base.distribution import Distribution
+
+
+class BackendDistribution(Distribution):
+    """
+    BackendDistribution for Tensorflow distributions
+    """
+    __doc__ = Distribution.__doc__ + __doc__
+
+    def sample(self, *args, **kwargs):
+        """
```

Comment:
that's why you may miss the docstring. If you leave it empty, it will be filled from inheritance